bridgesync/bridgesync.go

```diff
@@ -64,14 +64,22 @@ type BridgeSync struct {
 	bridgeContractV2 *polygonzkevmbridgev2.Polygonzkevmbridgev2
 }
 
+// noOpReorgDetectorWrapper wraps NoOpReorgDetector to implement bridgesync.ReorgDetector interface
+type noOpReorgDetectorWrapper struct {
+	reorgdetector.NoOpReorgDetector
+}
+
+func (w *noOpReorgDetectorWrapper) GetLastReorgEvent(ctx context.Context) (reorgdetector.ReorgEvent, error) {
```

Comment:
I think you don't need this wrapper since `reorgdetector.NoOpReorgDetector` already implements this function.